docs/sdkx_android/notifications.mdx

```diff
@@ -67,10 +67,11 @@ Enter your FCM key credentials per app, via the
 **Settings page > App listing in the left navigation > Scroll down to Push
 Notifications** settings section for the app.
 
-<Image src="/static/books/helpshiftx/add_fcm_token.png" width="full" />{" "}
+<Image src="/static/books/helpshiftx/add_fcm_http_v1.png" width="full" />{" "}
 
-For FCM users, the API key can be found at your
-[Firebase API Console](https://console.firebase.google.com/)
+For FCM users, the Auth key file can be found at your
+[Firebase API Console.](https://console.firebase.google.com/) 
+For more Information [Click Here.](https://firebase.google.com/docs/cloud-messaging/migrate-v1?hl=en&authuser=0#provide-credentials-manually)
```

Comment:
For more information, [click here](https://firebase.google.com/docs/cloud-messaging/migrate-v1?hl=en&authuser=0#provide-credentials-manually).

^ Just minor polishing for the casing. Please make this for all docs where applicable.